### DISCUSSION TITLE
Does offering an public github action (docker based) eat my free ci usage time?

### DISCUSSION THREAD

**Discussion by janderssonse** [community/community#26640] [Actions]
<p>When offering an github action to the public (docker based) is that usage of the image deducted from one private CI time, or is it infinite in this case? I presumed the last, but did not find a clear answer.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>A workflow (and any actions in it) eats the time of the one who <em>runs</em> it. So if you put it in a workflow for your repository, it consumes your time. If someone else uses the action in their repository, they have to feed it with their CI minutes. :wink:</p>
